Documentation/operations/upgrade.rst

```diff
@@ -315,7 +315,12 @@ Annotations:
   Upgrading from Cilium 1.14.x or earlier to 1.15.y or later does not
   trigger this problem. Downgrading from Cilium 1.15.y or later to Cilium
   1.14.x or earlier may trigger this problem.
-  
+
+Added Metrics
```

Comment:
There is a "Changed Metrics" section in the upgrade notes, can you move "Added Metrics" section to directly precede it?

Reply:
Ah, didn't see that other section. Moved.